styles.css

```diff
@@ -1,3 +1,30 @@
 .home {
   font-family: 'Roboto', sans-serif;
 }
+body {
+  margin: 0px;
+}
+.wrapper {
+  margin: auto;
+  width: 80%;
+}
+.header {
+  display:flex;
+  justify-content: space-between;
+}
+.logo, .menu {
+   display: inline-block;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6226565)